code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -153,6 +153,8 @@
 	if(!is_component_functioning("actuator"))
 		src.Paralyse(3)
 
+	if(ticker && ticker.mode.name == "nations")

```

Comment:
What @tigercat2000 means is that well.....essentially, Tiger refactored life() code to be like TG's; that is to say, pushing as much, as possible, up the life() chain---as such, this means far less copy-pasted code, ergo why it would be better to have `process_nations` be in living/life() (as that applies to humans, AI's, and cyborgs) as opposed to each life() proc.